src/Symfony/Bundle/FrameworkBundle/Resources/config/console.php

```diff
@@ -0,0 +1,290 @@
+        ->tag('kernel.event_subscriber')
+
+        ->set('console.command.about', AboutCommand::class)
+        ->tag('console.command', ['command' => 'about'])
+
+        ->set('console.command.assets_install', AssetsInstallCommand::class)
+        ->args([
+            service('filesystem'),
+            param('kernel.project_dir'),
+        ])
+        ->tag('console.command', ['command' => 'assets:install'])
+
+        ->set('console.command.cache_clear', CacheClearCommand::class)
+        ->args([
+            service('cache_clearer'),
+            service('filesystem'),
+        ])
+        ->tag('console.command', ['command' => 'cache:clear'])
+
+        ->set('console.command.cache_pool_clear', CachePoolClearCommand::class)
+        ->args([
+            service('cache.global_clearer'),
+        ])
+        ->tag('console.command', ['command' => 'cache:pool:clear'])
+
+        ->set('console.command.cache_pool_prune', CachePoolPruneCommand::class)
+        ->args([
+            [],
+        ])
+        ->tag('console.command', ['command' => 'cache:pool:prune'])
+
+        ->set('console.command.cache_pool_delete', CachePoolDeleteCommand::class)
+        ->args([
+            service('cache.global_clearer'),
+        ])
+        ->tag('console.command', ['command' => 'cache:pool:delete'])
+
+        ->set('console.command.cache_pool_list', CachePoolListCommand::class)
+        ->args([
+            null,
+        ])
+        ->tag('console.command', ['command' => 'cache:pool:list'])
+
+        ->set('console.command.cache_warmup', CacheWarmupCommand::class)
+        ->args([
+            service('cache_warmer'),
+        ])
+        ->tag('console.command', ['command' => 'cache:warmup'])
+
+        ->set('console.command.config_debug', ConfigDebugCommand::class)
+        ->tag('console.command', ['command' => 'debug:config'])
+
+        ->set('console.command.config_dump_reference', ConfigDumpReferenceCommand::class)
+        ->tag('console.command', ['command' => 'config:dump-reference'])
+
+        ->set('console.command.container_debug', ContainerDebugCommand::class)
+        ->tag('console.command', ['command' => 'debug:container'])
+
+        ->set('console.command.container_lint', ContainerLintCommand::class)
+        ->tag('console.command', ['command' => 'lint:container'])
+
+        ->set('console.command.debug_autowiring', DebugAutowiringCommand::class)
+        ->args([
+            null,
+            service('debug.file_link_formatter')->nullOnInvalid(),
+        ])
+        ->tag('console.command', ['command' => 'debug:autowiring'])
+
+        ->set('console.command.event_dispatcher_debug', EventDispatcherDebugCommand::class)
+        ->args([
+            service('event_dispatcher'),
+        ])
+        ->tag('console.command', ['command' => 'debug:event-dispatcher'])
+
+        ->set('console.command.messenger_consume_messages', ConsumeMessagesCommand::class)
+        ->args([
+            null, // Routable message bus
```

Comment:
Instead of `null` + "comment explaining this argument", we should use an `abstract_arg()` like in this example: https://github.com/symfony/symfony/pull/36778/files#diff-c9620897a07dd9bcada6305d3623c687R54

Reply:
Fixed that as well